src/opnsense/service/templates/OPNsense/Monit/+TARGETS

```diff
@@ -1,2 +1,3 @@
 monitrc:/usr/local/etc/monitrc
 rc.conf.d:/etc/rc.conf.d/monit
+00-example.conf:/usr/local/etc/monit.opnsense.d/00-example.conf
```

Comment:
Can we do this without the templates? Just add the proper git file directly

Reply:
I'm sorry but I don't get what you mean. Could you explain me what exactly are you expecting and where should this file be located then?

Reply:
remove this entry and move 00-example.conf to src/etc/monit.opnsense.d/00-example.conf and done